src/Link_TEST.cc

```diff
@@ -203,3 +205,61 @@ TEST(DOMLink, InvalidInertia)
   EXPECT_DOUBLE_EQ(3.4, link.Inertial().MassMatrix().OffDiagonalMoments().Z());
   EXPECT_FALSE(link.Inertial().MassMatrix().IsValid());
 }
+
+/////////////////////////////////////////////////
```

Comment:
I would put these tests in `test/integration/link_dom.cc` since that's where most of the `Load` functionality is tested.

Reply:
Done 78bd7730848e8c1c5fe9fd7d601003f97609333b